### PR TITLE
Immediately apply changed temperature unit

### PR DESCRIPTION
**Pull request by DL6ER**
Internal fix

@pi-hole/dashboard

### CODE REVIEW

File: header.php [hunk 1]
```diff
@@ -4,18 +4,43 @@
...
+        // Override temperature unit setting if it is changed via Settings page
+        if(isset($_POST["tempunit"]))
+        {
+            $temperatureunit = $_POST["tempunit"];
+        }
     }
     else
     {
-        $temperatureunit = "C";
+        $celsius = -273.16;
```
Comment (Mcat12) [inline header.php]: Why is $celsius set to -273.16? Perhaps set it to null and check if it's null later.
http://php.net/manual/en/language.types.null.php

Comment (DL6ER) [inline header.php]: Nothing can be colder than -273.15°C ;-)

Comment (Mcat12) [inline header.php]: Can you add that as a comment?

Comment (DL6ER) [inline header.php]: @Mcat12 Here you are

**Comment by Mcat12** [comment]
Approved

**Pull request merged**